addons/website_event/models/event.py

```diff
@@ -175,8 +171,5 @@ def _get_event_resource_urls(self, attendees):
             'details': self.name,
         })
         google_url = GOOGLE_CALENDAR_URL + params
-        params = werkzeug.url_encode({
-            'attendees': dict(('attendee_%s' % attendee, attendee) for attendee in attendees)
-        })
-        iCal_url = '/event/%s/ics/%s.ics?' % (slug(self), self.name) + params
+        iCal_url = '/event/%s/ics?%s' % (slug(self), params)
```

Comment:
attendees as params no more used

address_id is Falsy when online event, so it will crash

Reply:
attendees was only used in order to get the email address, which is not useful in this case, we just want to record an event in the calendar. (the email rendered was the one given by the attendee which means we don't lose useful information).

As for the online event, I don't think it will be a problem as you cannot get to this route in that case (maybe future improvement), currently you have to go through the registration_confirm route.